Billing/billing-process-guidelines.md

```diff
@@ -0,0 +1,33 @@
+{{{
+  "title": "Ecosystem Partner Billing Setup Process",
```

Comment:
does this really belong in a public forum?  this seems like a bad place for it as I don't see how this helps any customer of the platform.  it seems to be a guide for internal processes in sales/ecosystem.  we have places for this like cloudhelp.ctl.io .  just want to ask as it reads pretty out there as a customer looking in the kb.  even the fact we ignore bad debt being stated public seems off.